DigitalLearningSolutions.Web/Views/RegisterAtNewCentre/Confirmation.cshtml

```diff
@@ -21,12 +23,20 @@
       <p>Your delegate number is <span class="nhsuk-u-font-weight-bold">@Model.CandidateNumber</span></p>
     </div>
 
+    <partial name="_VerifyEmailWarningForRegistrationConfirmationPage" model="@verifyEmailWarningViewModel" />

```

Comment:
Looks like this will always appear.

Reply:
Shouldn't assume that we'll have to show this on internal journeys.

Reply:
added a conditional